tests/plenary/ui/mappings/headline_spec.lua

```diff
@@ -90,26 +91,52 @@ describe('Heading mappings', function()
       'Content Level 3',
     }, vim.api.nvim_buf_get_lines(0, 2, 8, false))
     vim.fn.cursor(3, 1)
+    local check
+    if VirtualIndent.enabled then
+      check = {
+        '** TODO Test orgmode',
+        'DEADLINE: <2021-07-21 Wed 22:02>',
+        '*** TODO [#A] Test orgmode level 2 :PRIVATE:',
+        'Some content for level 2',
+        '**** NEXT [#1] Level 3',
+        'Content Level 3',
+      }
+    else
+      check = {
+        '** TODO Test orgmode',
+        '   DEADLINE: <2021-07-21 Wed 22:02>',
+        '*** TODO [#A] Test orgmode level 2 :PRIVATE:',
+        ' Some content for level 2',
+        '**** NEXT [#1] Level 3',
+        ' Content Level 3',
+      }
+    end
     vim.cmd([[norm >s]])
-    assert.are.same({
-      '** TODO Test orgmode',
-      '   DEADLINE: <2021-07-21 Wed 22:02>',
-      '*** TODO [#A] Test orgmode level 2 :PRIVATE:',
-      ' Some content for level 2',
-      '**** NEXT [#1] Level 3',
-      ' Content Level 3',
-    }, vim.api.nvim_buf_get_lines(0, 2, 8, false))
+    assert.are.same(check, vim.api.nvim_buf_get_lines(0, 2, 8, false))
 
     -- Support count
+    local check
+    if VirtualIndent.enabled then
```

Comment:
This is done to make tests pass on 0.9.* and 0.10 ?

Reply:
Yes, and I'll refactor it to not check that variable and instead rely on the buffer option (depending on your decision to [this comment](https://github.com/nvim-orgmode/orgmode/pull/627#discussion_r1465601412)) in combination with `org_adapt_indentation`.

Reply:
Refactored.